docs/getting-started.md

```diff
@@ -0,0 +1,89 @@
+
+## Step 2. Configure your store
+
+Virto Commerce platform is 4-MULTI: multi-language, multi-currency, multi-theme and multi-store by design. This gives the ability to have and run multiple stores on the same system.
+
+To configure your store you must accomplish a set of actions:
+
+1. Create first store
+2. Select your catalog
+3. Select available currency, languages
+4. Upload theme
+5. …
+6. Rebuild catalog index
+
+Check that new store is visible.
+
+The [Managing stores](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/configuration/store.md) article helps you to configure your first store. Also at this step you should configure [Users, Roles and Role Assignment](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/users-management-roles-and-role-assignment.md)
+
+## Step 3. Configure prices and inventory
+
+In the Virto Commerce, a price list is a set of pricing details that can be assigned to items. All prices are stored in a price list, which is associated with a catalog and a currency. When the price list is created in the Virto Commerce, you need to define which products will be included into this price list and what prices will be defined for them. You can do it by creating price list assignment.
+
+Price list assignment identifies the price list from which a product price is shown for customers. Moreover, price list assignments can do much more than simply provide default list and sale prices for products. This feature allows you to provide custom prices to targeted customers according to customer's characteristics such as age, gender, geographic location and searched terms.
+
+After this step completed, you can [Open new products in Virto Commerce Storefront](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/lessons/lesson2.md#open-new-products-in-virto-commerce-storefront-frontend) , browse the catalog, add product to shopping cart and create order. You can view customer order via admin UI.
+
+## Step 4. Configure payments, shipments, taxes
+
+By default, you see demo payment and shipmen methods. Virto Commerce provides the ready to use module which. You can set your own configurations:
+
+* Configure [Payments](TODO) method
+* Configure [Shipments](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/order-management/working-with-shipments.md) method
+* Configure [Taxes](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/order-management/working-with-taxes.md) method
+
+## Step 5. Launch Marketing & promotional tools
+
+Marketing your store is essential to gain your customers loyalty and raise awareness among potential customers.
+
+There are several marketing tools in Virto Commerce: you can publish content, create personalized shopping experiences for customers, manage promotions.
+
+Start with Creation of two simple promotions:
+
+1. 5% if cart subtotal > 100 $
```

Comment:
Add more details: Added  more details

Reply:
Didn't get what needs to be done, need more details :)

Reply:
Could it be: "Each customer would receive a 5 percent off if cart subtotal exceeded $100."?
Add more details to the second promotion as well.

Reply:
fixed